electron/src/lib/openGraph.ts

```diff
@@ -58,18 +58,16 @@ const fetchOpenGraphData = (url: string): Promise<OpenGraphResult> => {
   const parsedUrl = urlUtil.parse(url);
   const normalizedUrl = parsedUrl.protocol ? parsedUrl : urlUtil.parse(`http://${url}`);
 
-  const parseHead = (body: string) => {
-    const [head] = body.match(/<head>[\s\S]*?<\/head>/) || [''];
-    return openGraphParse(head);
-  };
-
   return new Promise<string>((resolve, reject) => {
-    const getContentRequest = request.get(urlUtil.format(normalizedUrl), (error, response, body: string) => {
-      return error ? reject(error) : resolve(body);
-    });
+    let partialBody = '';
+    const getContentRequest = request.get(urlUtil.format(normalizedUrl));
+
+    getContentRequest.on('response', response => {
+      if (response.statusCode !== 200) {
```

Comment:
How should we handle redirects?

Reply:
I don't think we should trigger an error for redirects.
If you link `website.com` and you are redirected to a valid `www.website.com` you would not be please if the link preview doesn't show ... I guess.

Reply:
Seems like the `statusCode` is checked after the redirect.